scripts/migration/lib.sh

```diff
@@ -215,7 +218,7 @@ if [ -z "${CK8S_ROOT_SCRIPT:-}" ]; then
     export CK8S_STACK="${CK8S_STACK:-}/${THIS}"
   fi
 
-  check_config
+  check_config "${1:-both}"
```

Comment:
I don't think `$1` will ever be set here, since `lib.sh` only is sourced and not executed as a regular script. You would have to handle checking only sc or wc config in some other way, but I'm not sure what the easiest solution for that would be.

Reply:
@Ajarmar How about having an environment variable which is exported in `upgrade.bash` and used in `lib.sh` (and if not set defaults to `both`)? See latest commit [here](https://github.com/elastisys/compliantkubernetes-kubespray/pull/317/commits/c71b1458edf564aa98bed2fc76a658d6c0cc37b2#diff-7da0a7c55357aa1013e4acc3d29497da33350bcec142c561e57ecbc170a45bd3R9) and [here](https://github.com/elastisys/compliantkubernetes-kubespray/pull/317/commits/c71b1458edf564aa98bed2fc76a658d6c0cc37b2#diff-d157d0f7a32fe5a99591a42778e449309300204efe637caef726eefdceb9f0deR223).